packages/shared/src/modules/users/libs/validation-schemas/user-get-profile.validation-schema.ts

```diff
@@ -0,0 +1,12 @@
+import { z } from "zod";
+
+import { UserValidationMessage } from "../enums/enums.js";
+
+const userGetProfile = z.object({
```

Comment:
if this is a parementer validation, let's clarify this
```suggestion
const userGetProfileParameters = z.object({
```